### ISSUE TITLE
Exception when run against: `https://github.com/gitlabhq/gitlabhq`

### ISSUE THREAD

**Issue by mscottford** [corgibytes/freshli#35]
Exception from Jupyter Notebook:

```
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Where[TSource](IEnumerable`1 source, Func`2 predicate)
   at Freshli.Languages.Ruby.RubyGemsRepository.LatestAsOf(DateTime date, String name)
   at Freshli.LibYearCalculator.ComputeAsOf(DateTime date)
   at Freshli.Runner.Run(String analysisPath, DateTime asOf)
   at Freshli.Runner.Run(String analysisPath)
   at Submission#19.<<Initialize>>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.CodeAnalysis.Scripting.ScriptExecutionState.RunSubmissionsAsync[TResult](ImmutableArray`1 precedingExecutors, Func`2 currentExecutor, StrongBox`1 exceptionHolderOpt, Func`2 catchExceptionOpt, CancellationToken cancellationToken)
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Where[TSource](IEnumerable`1 source, Func`2 predicate)
   at Freshli.Languages.Ruby.RubyGemsRepository.LatestAsOf(DateTime date, String name)
   at Freshli.LibYearCalculator.ComputeAsOf(DateTime date)
   at Freshli.Runner.Run(String analysisPath, DateTime asOf)
   at Freshli.Runner.Run(String analysisPath)
   at Submission#19.<<Initialize>>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.CodeAnalysis.Scripting.ScriptExecutionState.RunSubmissionsAsync[TResult](ImmutableArray`1 precedingExecutors, Func`2 currentExecutor, StrongBox`1 exceptionHolderOpt, Func`2 catchExceptionOpt, CancellationToken cancellationToken)
```

**Comment by nickiemc** [comment]
The exception above has been addressed. 

Running against the latest code, a number of errors occur and a number of dependencies are skipped as seen in the [attached output](https://github.com/corgibytes/freshli/files/5315878/2020-10-01-gitlabhq-output.txt).

**Comment by nickiemc** [comment]
The remaining issues will be resolved by #153. Marking as closed.
